### ISSUE TITLE
How do I uninstall Aftman?

### ISSUE THREAD

**Issue by starrstrux** [LPGhatguy/aftman#49]
I'm probably stupid but I can't figure out how to install Aftman as I prefer Foreman, but I don't know how to just get rid of it off my computer and all projects. It's not like Aftman is installing Rojo anyway so I might as well use Foreman. Anyone that can help?

**Comment by MacaylaMarvelous81** [comment]
Have you checked .aftman in your HOME? (~/.aftman or %USERPROFILE%/.aftman)

**Comment by Bobnest** [comment]
> Have you checked .aftman in your HOME? (~/.aftman or %USERPROFILE%/.aftman)

So do we just delete the folder from there?

**Comment by LastTalon** [comment]
Other than the configuration aftman does for your path, the entire installation is contained in that folder, so removing it should remove aftman.

**Comment by setsuw** [comment]
does this also remove tools installed by aftman? if not, how would i go about doing that?

**Comment by OverHash** [comment]
> does this also remove tools installed by aftman? if not, how would i go about doing that?

Yes, aftman stores all its tools under `~/.aftman` (or appropriate alternative on other operating systems).

**Comment by LPGhatguy** [comment]
Aftman is being archived. Please see LPGhatguy/aftman#74 for more information and alternatives!
